src/main/java/survey/Question.java

```diff
@@ -13,7 +13,7 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    protected long id;
+    protected int id;
```

Comment:
why are we changing it from a long to an int?

Reply:
In javascript numbers are represented by floating point numbers. That means that for sufficiently large numbers integers cannot be represented even if written into a number field. The string number will be converted to a slightly different floating point number. The range when this starts to happen is for number well above 2^32. To make sure users never encounter the problem of their numerical responses being changed, we force the results to always be an integer (i.e. between -2^31 and 2^31 - 1).

Reply:
For this field, if it were an id, and it were transmitted to the user, the user could never retrieve their survey cause they couldn't represent the number exactly as an actual number type.